box/src/main/scala/blobstore/box/BoxStore.scala

```diff
@@ -222,9 +222,11 @@ final class BoxStore[F[_]](
 object BoxStore {
   def apply[F[_]](
     api: BoxAPIConnection,
-    rootFolderId: String,
-    blocker: Blocker
-  )(implicit F: Concurrent[F], CS: ContextShift[F]): BoxStore[F] = new BoxStore(api, rootFolderId, blocker)
+    blocker: Blocker,
+    rootFolderId: String = RootFolderId
+  )(implicit F: Concurrent[F], CS: ContextShift[F]): BoxStore[F] = new BoxStore(api, blocker, rootFolderId)
+
+  val RootFolderId = "0"
```

Comment:
Is this needed?

Reply:
Or is it being resolved to root folder?

Reply:
Yes, defaulting it to root if it's not set by the user. Changed the order of arguments to be consistent with the apply method where this is last due to a default parameter.